src/coq/logicutils/inductive/indutils.ml

```diff
@@ -146,16 +146,6 @@ let arity_of_ind_body ind_body =
     let sort = Constr.mkType template_level in
     recompose_prod_assum ind_body.mind_arity_ctxt sort
 
-(* Create an Entries.local_entry from a Rel.Declaration.t *)
-let make_ind_local_entry decl =
```

Comment:
What happened to this?

Reply:
The type of mind_entry_params changed, which was the only place we were using this function